docs/automation-and-resiliency/cicd-pipeline-templates-for-private-cloud-teams.md

```diff
@@ -0,0 +1,78 @@
+---
+title: CI/CD pipeline templates for Private Cloud teams
+
+slug: ci-cd-pipeline-templates
+
+description: Describes continuous integration (CI) and continuous delivery (CD) pipeline automation and how teams can use it in their applications.
+
+keywords: CI, CD, application, application tools, pipelines, pipeline templates
+
+page_purpose: Gives an outline of pipeline automation and templates for teams that want to use them or contribute to them.
+
+audience: developer, technical lead
+
+author: Jonathan Bond
+
+content_owner:
+---
+
+# CI/CD pipeline templates for Private Cloud teams
+
+Once you've built a new application and want to deploy it to the BC Government's Private Cloud OpenShift platform, you might also want to use continuous integration (CI) and continuous delivery (CD) power. This lets your team automatically build, test, and prepare code changes for production so that software delivery is more efficient and secure.
```

Comment:
Minor detail: Teams should create pipelines from the start of development.  These new templates show how to augment their pipelines with tests.  We might reword this as:

"As you develop your application for deployment to the BC Government's Private Cloud OpenShift platform, you should create a pipeline that automatically builds and tests your code so that your software delivery is efficient and secure.  Use our pipeline templates to help you get started."